test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -44,5 +48,87 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace("script processed queue message")));
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace(messageContent)));
         }
+
+        protected async Task EasyTablesTest(bool writeToQueue = true)
```

Comment:
The intent for this base class was that anything added here runs against ALL languages, since they're inherited. This protected member allows you to only test C#/Node? That might be OK (we do have a large test matrix for all languages :)). Have you tried other languages?